Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+pipeline {
+    agent any
+    stages {
+        stage('Verify git commit conventions') {
+            steps {
+                sh 'sh ./gitlint.sh'
+            }
+        }
+        stage('Verify Format') {
+            steps {
+                sh 'docker build --target verify-format .'
+            }
+        }
+        stage('Verify .sh files') {
+            steps {
+                sh 'docker build --target verify-sh .'
+            }
+        }
```

Comment:
I think that these stages could be unified in only one like

```
        stage('Verify') {
            steps {
                sh 'echo Verify git commit conventions'
                sh 'sh ./gitlint.sh'
                sh 'echo Verify Format'
                sh 'docker build --target verify-format .'
                sh 'echo Verify .sh files'
                sh 'docker build --target verify-sh .'
            }
        }
```

or has stage of stages

Reply:
It depends on the granularity that you want to have in Jenkins UI.